### PR TITLE
IGAPP-902: Poi details view web desktop

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

http://localhost:9000/testumgebung/de/locations
1. Test select & deselect pois on the map
2. select / deselect on List / Details View (goBack)
3. Switch pois in the detail navigation panel
4. Test link with queryParams (http://localhost:9000/testumgebung/de/locations?name=contact-sozialkaufhaus) The short loading of the `no pois found` and `list view` gonna be fixed in the refactoring pr

**Note:** Tests gonna be added in refactoring pr, because i don't want to get on pr even larger

### CODE REVIEW

File: build-configs/common/theme/colors.ts [hunk 1]
```diff
@@ -18,5 +19,6 @@ export const commonLightColors = {
   textDisabledColor: '#d0d0d0',
   backgroundColor: '#ffffff',
   tunewsThemeColor: '#007aa8',
-  tunewsThemeColorLight: 'rgba(0, 122, 168, 0.4)'
+  tunewsThemeColorLight: 'rgba(0, 122, 168, 0.4)',
+  poiBorderColor: '#F1F1F1'
```
Comment (klinzo) [inline build-configs/common/theme/colors.ts:23]: We really need another color?

File: translations/translations.json [hunk 2]
```diff
@@ -4860,7 +4860,13 @@
       "phoneCopied": "Telefonnummer in Zwischenablage kopiert",
       "eMailCopied": "E-Mail-Adresse in Zwischenablage kopiert",
       "websiteCopied": "Webseite in Zwischenablage kopiert",
-      "listTitle": "In deiner Nähe"
+      "listTitle": "In deiner Nähe",
+      "detailsHeader": "zurück zu Übersicht",
```
Comment (klinzo) [inline translations/translations.json:4864]:
```suggestion
      "detailsHeader": "zurück zur Übersicht",
```

File: translations/translations.json [hunk 3]
```diff
@@ -4860,7 +4860,13 @@
...
+      "detailsAddress": "Wegbeschreibung",
+      "detailsMapLink": "In Google Maps öffnen",
+      "detailsInformation": "Informationen",
+      "detailsNextPoi": "nächstes",
+      "detailsPreviousPoi": "vorheriges"
```
Comment (klinzo) [inline translations/translations.json:4869]: I think it should be the following as it is `Der Ort`
```suggestion
      "detailsNextPoi": "nächster",
      "detailsPreviousPoi": "vorheriger"
```

File: translations/translations.json (hunk 2 above)
Comment (klinzo) [reply translations/translations.json:4864]: Does `detailsXYZ` prefix add any additional information?

File: web/src/routes/__tests__/PoisPage.spec.tsx [hunk 4]
```diff
@@ -33,11 +35,10 @@ describe('PoisPage', () => {
...
   const poi1 = pois[1]!
 
-  const routePattern = `/:cityCode/:languageCode/${RoutePatterns[POIS_ROUTE]}`
+  const pathname = cityContentPath({ route: POIS_ROUTE, cityCode: city.code, languageCode: language.code })
 
-  const renderPois = ({ id }: { id?: string } = {}) => {
-    const pathname = cityContentPath({ route: POIS_ROUTE, cityCode: city.code, languageCode: language.code, path: id })
-    return renderRoute(
+  const renderPois = (
+    <ThemeProvider theme={buildConfig().lightTheme}>
```
Comment (klinzo) [inline web/src/routes/__tests__/PoisPage.spec.tsx:41]: I think I fixed this already but please use the utility render method and just pass `{ wrapWithTheme: true }` as second argument

File: web/src/routes/__tests__/PoisPage.spec.tsx [hunk 5]
```diff
@@ -1,21 +1,24 @@
...
   ...jest.requireActual('api-client'),
   useLoadFromEndpoint: jest.fn()
 }))
-jest.mock('react-i18next')
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key
+  })
+}))
```
Comment (klinzo) [inline web/src/routes/__tests__/PoisPage.spec.tsx:21]: I think I already fixed this but `jest.mock('react-i18next')` is enough (we have a custom mock here)

File: web/src/constants/dimensions.ts [hunk 6]
```diff
@@ -19,7 +19,7 @@ const dimensions: DimensionsType = {
   minMaxWidth: 'screen and (min-width: 1100px)',
   maxWidth: 1100,
   toolbarWidth: 125,
-  toolbarHeight: 73,
+  toolbarHeight: 44,
```
Comment (klinzo) [inline web/src/constants/dimensions.ts:22]: Why did you change this?

File: web/src/components/PoiDetails.tsx [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+  }
+
+  const { title, thumbnail, distance } = feature.properties
+  const { content, location } = poi
+  const { t } = useTranslation('pois')
+  const navigate = useNavigate()
+  // MapEvent parses null to 'null'
+  const thumb = thumbnail === 'null' ? null : thumbnail?.replace('-150x150', '')
+  return (
+    <DetailsContainer>
```
Comment (klinzo) [inline web/src/components/PoiDetails.tsx:130]: In a future PR we should probably make the sections resuable, fine for now thoguh

**Review by klinzo — APPROVED** [review]
Fine for me together with #646

File: web/src/constants/dimensions.ts (hunk 6 above)
Comment (f1sh1918) [reply web/src/constants/dimensions.ts:22]: because i decreased the toolbar height (some icon padding). it was not necessary to have such a big height

File: web/src/routes/__tests__/PoisPage.spec.tsx (hunk 5 above)
Comment (f1sh1918) [reply web/src/routes/__tests__/PoisPage.spec.tsx:21]: gonna be fixed in your pr so?

File: web/src/routes/__tests__/PoisPage.spec.tsx (hunk 4 above)
Comment (f1sh1918) [reply web/src/routes/__tests__/PoisPage.spec.tsx:41]: this gonna be also fixed in your pr

File: build-configs/common/theme/colors.ts (hunk 1 above)
Comment (f1sh1918) [reply build-configs/common/theme/colors.ts:23]: we don't have a fitting color not even close to the design. Quite too darken and the separation looks ugly. so i would say yes :)

File: web/src/components/PoiDetails.tsx (hunk 7 above)
Comment (f1sh1918) [reply web/src/components/PoiDetails.tsx:130]: if you check the poi details screen i think there is a lot of code needed for styling to make it reusable. i don't see the need at the moment. if we use some similar component on other screen. sure it does

File: web/src/routes/__tests__/PoisPage.spec.tsx (hunk 4 above)
Comment (klinzo) [reply web/src/routes/__tests__/PoisPage.spec.tsx:41]: yeah thats what I wrote, just wanted to let you know :D

File: web/src/routes/__tests__/PoisPage.spec.tsx (hunk 5 above)
Comment (klinzo) [reply web/src/routes/__tests__/PoisPage.spec.tsx:21]: just wanted to let you know for the next time :)

File: web/src/components/PoiDetails.tsx (hunk 7 above)
Comment (klinzo) [reply web/src/components/PoiDetails.tsx:130]: I meant the separate sections like opening hours, address, ...

**Review by SteffiStuffel — APPROVED** [review]
Not reviewed, just tested: Found just one thing: When using the "next"-Button in the Footer I get the next POI in alphabetical order, nicer would be to get the next ordered by distance (so it has the same order as in the list too)
